### PR TITLE
feat: (KW-144) Allow user to view Monthly filtered transactions

### PR DESCRIPTION
**Pull request by shottah**
### Description

_A few sentences describing the overall effects and goals of the pull request's commits.
What is the current behavior, and what is the updated/expected behavior with this PR?_

Clicking on the right arrow at the section header for each group of transactions in the Wallet Home transaction feed will take you to a static set of transactions filtered for that month. The navigation bar has an "Export" button which opens a prompt for the user to export their transactions.

### Other changes

_Describe any minor or "drive-by" changes here._

Code was written to add in multiple filters, which will be completed in a further work item.
* Month
* Amount
* Recipient

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### How others should test

_Does this need to be tested by QA in the next release cycle? If so please give a brief explanation of how to test these changes._

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

**Review by Nashmod — COMMENTED** [review]
Both on my IOS and Android tests I'm not actually seeing the faucet or transaction in the recent filter section. If that wasn't supposed to be implemented yet then the rest looks okay but in the code i can see that there should be something.

Also how do I get the other filter types? I can only see the recent filter.

image

image

File: src/transactions/utils.ts [hunk 1]
```diff
@@ -95,3 +103,55 @@ export const transferReviewHeader = (
...
+export const filterTxByMonth = (
+  transactions: FeedTokenTransaction[],
+  month: string | undefined
+) => {
+  if (!month) return transactions
+  return filter(transactions, (tx) => {
+    const target = formatFeedSectionTitle(tx.timestamp, i18n)
+    return target === month
+  })
+}
```
Comment (shottah) [inline src/transactions/utils.ts:119]: @Nashmod okay, seems that the screen is navigated with the filter 'Recent', but when the transactions are filtered, they are marked as 'August' so they get filtered out. I will take a look at this.

Comment (Nashmod) [reply src/transactions/utils.ts:119]: Sounds good! Will wait on that update then.

**Comment by shottah** [comment]
@Nashmod ready for review.

**Pull request merged**